platform/security/handler/pom.xml

```diff
@@ -28,6 +30,16 @@
             <groupId>org.apache.shiro</groupId>
             <artifactId>shiro-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.platform</groupId>
```

Comment:
Shouldn't need to add this dependency. There should be no classes from an implementation bundle that any consumers need to know about; they should be interacting through interfaces defined in the API.